moz_telemetry/modules/moz_telemetry/extract_dimensions.lua

```diff
@@ -60,7 +60,11 @@ local schema_path   = read_config("schema_path") or error("schema_path must be s
 local content_field = read_config("content_field") or "Fields[content]"
 local uri_field = read_config("content_field") or "Fields[uri]"
 
-local city_db = assert(geoip.open(read_config("geoip_city_db")))
+local city_db_file = read_config("geoip_city_db")
```

Comment:
You can actually make the inclusion of the entire module optional.  i.e. if you aren't using geoip decoding you don't even have to have load the `.so`